### PR TITLE
edgehub dispatch message by type

### PR DESCRIPTION
**Pull request by lvchenggang**
What type of PR is this?
/kind feature

What this PR does / why we need it:
Like the previous way, through array to dispatch message in edgehub is not intuitive, by switch-case grammar will be clearer and maintainable.

Does this PR introduce a user-facing change?:
NONE

### CODE REVIEW

File: edge/pkg/edgehub/process.go [hunk 1]
```diff
@@ -81,11 +81,20 @@ func (eh *EdgeHub) sendToKeepChannel(message model.Message) error {
...
+		md = modules.MetaGroup
+	case messagepkg.TwinGroupName:
+		md = modules.TwinGroup
+	case messagepkg.FuncGroupName:
+		md = modules.MetaGroup
+	case messagepkg.UserGroupName:
+		md = modules.BusGroup
+	default:
+		klog.Warningf("msg_group: %s not found", group)
+		return fmt.Errorf("msg_group: %s not found", group)
```
Comment (fisherxu) [inline edge/pkg/edgehub/process.go:97]: What's the difference of these two implementation?

Comment (fisherxu) [reply edge/pkg/edgehub/process.go:97]: Maybe the groupMap can be deleted?

Comment (lvchenggang) [reply edge/pkg/edgehub/process.go:97]: Use switch-case looks more intuitive, the groupMap structure still  used in edgehub currently.

**Comment by fisherxu** [comment]
Looks like there's something wrong with the CI

**Comment by lvchenggang** [comment]
> Looks like there's something wrong with the CI/UT

Yes, I will look at this question.

**Comment by lvchenggang** [comment]
> > Looks like there's something wrong with the CI/UT
> 
> Yes, I will look at this question.

The UT problem has been resolved.

**Comment by fisherxu** [comment]
> > > Looks like there's something wrong with the CI/UT
> > 
> > 
> > Yes, I will look at this question.
> 
> The UT problem has been resolved.

How does the utility be fixed?  Looks like has no modification ;-)

**Comment by lvchenggang** [comment]
> > > > Looks like there's something wrong with the CI/UT
> > > 
> > > 
> > > Yes, I will look at this question.
> > 
> > 
> > The UT problem has been resolved.
> 
> How does the utility be fixed? Looks like has no modification ;-)

The UT error is caused by the following changes:
image

Finally, I reducing these changes and everything is OK

**Pull request merged**